packages/node/src/requestdata.ts

```diff
@@ -310,8 +310,8 @@ function extractQueryParams(req: PolymorphicRequest): string | Record<string, un
 
   return (
     req.query ||
-    (typeof URL !== undefined && new URL(originalUrl).search.replace('?', '')) ||
```

Comment:
> On node v8 `URL` would have always been `undefined`

Yup - that's what the `url.parse()` line is there for.

The reason there's the bare `URL` is because for a hot second this code lived in utils and therefore needed to be compatible with the browser as well. Given that if any part of it ever ends up back in the land of shared code it'll be this part, I'm tempted to say we should just leave this be. (Unless is straight-up broken, but I don't believe that's the case, is it?)

Reply:
> Yup - that's what the url.parse() line is there for.

🤦🏻‍♂️

Looks like a version of this code still lives in utils with an `InjectedNodeDeps`:
https://github.com/getsentry/sentry-javascript/blob/12c19d051eca4fcef316ba6ee512f1221ae92ef0/packages/utils/src/requestdata.ts#L339-L342

Given that we're not supporting node v8 for much longer this change is almost certainly gratuitous but after removing dom types, TypeScript was throwing a build error!